services/src/main/java/io/scalecube/services/ServiceTransport.java

```diff
@@ -23,9 +24,8 @@ public ServiceTransport(Transport transport) {
 
   @Override
   public CompletableFuture<Void> send(Address address, Message message) {
-    CompletableFuture<Void> future = new CompletableFuture<Void>();
-    transport.send(address, message, future);
-    return future;
+    transport.send(address, message, COMPLETED_PROMISE);
+    return COMPLETED_PROMISE;
```

Comment:
I don't see that it makes much value to return anything at all if it is always completed promise. In such case it is probably better to just say that send don't have any return value, just say it returns `void`. Also transport has send method which do not accept any parameter which practically doing this same thing